packages/terra-clinical-detail-view/package.json

```diff
@@ -21,7 +21,7 @@
   },
   "homepage": "https://github.com/cerner/terra-clinical#readme",
   "peerDependencies": {
-    "react": "^16.4.2",
+    "react": "^16.8.5",
     "react-dom": "^16.4.2"
```

Comment:
Wanna get this feller?

Reply:
https://github.com/cerner/terra-clinical/pull/476/commits/ca2253ebee49b5a308b5906f148ffb7a1af57ee7